backend/src/sms/services/twilio-client.class.ts

```diff
@@ -11,17 +11,26 @@ export default class TwilioClient {
     this.messagingServiceSid = messagingServiceSid
   }
 
-  public send(recipient: string, message: string): Promise<string | void> {
+  public send(
+    recipient: string,
+    message: string,
+    forceDelivery = false
+  ): Promise<string | void> {
     return this.client.messages
       .create({
         to: recipient,
         body: this.replaceNewLines(message),
         from: this.messagingServiceSid,
+        forceDelivery: forceDelivery,
       })
       .then((result: { [key: string]: string }) => {
         const { status, sid, error_code: errorCode, code } = result
```

Comment:
Tested this through the settings page and the force delivery doesn't seem to be triggered.

Looks like when the validation fails on Twilio's side, the promise from `this.client.messages.create(...)` doesn't resolve and is [rejected](https://github.com/twilio/twilio-node/blob/abf3470d89c68f5efbb6eeb811242404ec59762d/lib/rest/api/v2010/account/message.js#L135) instead.